packages/account/src/Components/poa/status/needs-review/needs-review.tsx

```diff
@@ -3,9 +3,14 @@ import { localize } from '@deriv/translations';
 import React from 'react';
 import IconMessageContent from 'Components/icon-message-content';
 import PoiButton from 'Components/poi/poi-button';
-import { ContinueTradingButton } from 'Components/poa/continue-trading-button/continue-trading-button.jsx';
+import { ContinueTradingButton } from 'Components/poa/continue-trading-button/continue-trading-button';
 
-export const NeedsReview = ({ needs_poi, is_description_enabled = true }) => {
+type TNeedsReview = {
```

Comment:
```suggestion
type TNeedsReview = Record<'needs_poi' | 'is_description_enabled', boolean>
```

Reply:
@niloo-fs how to set needs_poi as a required and is_description_enabled as optional?

Reply:
My Bad, I just wanted to use `Record` here